tests/get_search.test.ts

```diff
@@ -473,7 +473,19 @@ describe.each([
       .index(emptyIndex.uid)
       .searchGet('', { vector: [1], hybridSemanticRatio: 1.0 })
 
-    expect(response.vector).toEqual([1])
+    expect(response).toHaveProperty('hits')
+    expect(response).toHaveProperty('semanticHitCount')
+    // Those fields are no longer returned by the search response
+    // We want to ensure that they don't appear in it anymore
+    expect(response).not.toHaveProperty('vector')
+    expect(response).not.toHaveProperty('_semanticScore')
+  })
+
+  test(`${permission} key: search without vectors`, async () => {
+    const client = await getClient(permission)
+    const response = await client.index(index.uid).search('prince', {})
+
+    expect(response).not.toHaveProperty('semanticHitCount')
```

Comment:
Creation of this specific test to make sure that with a "non-vector" search, I don't retrieve the `semanticHitCount` field in the response